sympy/physics/vector/tests/test_frame.py

```diff
@@ -454,6 +454,8 @@ def test_orient_axis():
     A = ReferenceFrame('A')
     B = ReferenceFrame('B')
     assert A.orient_axis(B,-B.x, 1) == A.orient_axis(B, B.x, -1)
+    assert A.orient_axis(B,-B.x, 1) == A.orient_axis(B, 1, -B.x)
```

Comment:
These two functions do not return anything, so the result here is `assert None == None`, right? I don't think this is what you are hoping for in this test.

Reply:
I corrected that